apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/data_updater/impl.ex

```diff
@@ -1727,6 +1768,40 @@ defmodule Astarte.DataUpdaterPlant.DataUpdater.Impl do
     {:discard, :unexpected_control_message, new_state, {:continue, continue_arg}}
   end
 
+  defp decode_bson_payload(state, payload) do
+    with {:error, error} <- Cyanide.decode(payload) do
+      Logger.warning(
+        "error parsing capabilities BSON message: #{error}",
+        tag: "capabilities_error"
```

Comment:
Another example of tag tha could be more informative:
```suggestion
        tag: "invalid_capabilities_payload"
```